tests/test_habitat_locobot.py

```diff
@@ -26,27 +28,167 @@ def create_robot():
     return bot
 
 
+def test_camera_reset(create_robot):
+    bot = create_robot
+    bot.camera.reset()
+    assert np.sum(np.abs(bot.camera.get_state())) < 1e-3
+
+
 @pytest.mark.parametrize(
     "target_position", [[0, 0.7], [0.4, 0.4], [0.4, -0.4], [-0.4, 0.4], [-0.4, -0.4]]
 )
 def test_camera_position_control(create_robot, target_position):
+    thr = 1e-3
     bot = create_robot
     bot.camera.reset()
     bot.camera.set_pan_tilt(target_position[0], target_position[1], wait=True)
-    achieved_position = bot.camera.get_state()
-    ag_error = np.fabs(np.array(achieved_position) - np.array(target_position))
-    assert np.max(ag_error) < 0.10
+    assert np.sum(np.abs(np.array(bot.camera.get_state()) - np.array(target_position))) < thr
```

Comment:
you can also use something like `np.allclose` if it's better: https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

Reply:
@soumith thanks for recommending `np.allclose`. I think its more suitable. I have updated the PR based on it

Reply:
great, the changes look good